packages/grafana-data/src/types/panel.ts

```diff
@@ -190,6 +228,16 @@ export class PanelPlugin<TOptions = any> extends GrafanaPlugin<PanelPluginMeta>
   }
 }
 
+export interface PanelOptionsEditorProps<TValue> extends StandardEditorProps<TValue> {}
+
+export interface PanelOptionsEditorItem<TValue = any, TSettings = any> extends RegistryItem {
+  editor: ComponentType<PanelOptionsEditorProps<TValue>>;
+  settings?: TSettings;
+}
+
+export interface PanelOptionsEditorConfig<TSettings = any, TValue = any>
+  extends Pick<PanelOptionsEditorItem<TValue, TSettings>, 'id' | 'description' | 'name' | 'settings'> {}
```

Comment:
Not sure I follow this (the Pick), what is omitted here?

Reply:
It's the `editor` prop from `PanelOptionsEditorItem` interface ant the entire irrelevand shebang from `RegistryItem` interface